lib/p2p/Pool.ts

```diff
@@ -712,18 +721,14 @@ class Pool extends EventEmitter {
         reject(err);
       };
 
-      this.server!.listen(this.listenPort || 0, '0.0.0.0').on('listening', () => {
```

Comment:
The comment block for the `listen` method should be updated to specify that multiple interfaces can be used.